website/project/views/contributor.py

```diff
@@ -292,7 +292,7 @@ def project_manage_contributors(auth, node, **kwargs):
         )
         if node.is_public:
             return {'redirectUrl': node.url}
-        return {'redirectUrl': web_url_for('dashboard')}
+        return {'redirectUrl': web_url_for('my_projects')}
```

Comment:
Sara agrees with you. :) This should have been changed to the dashboard when we added the new dashboard. I expect test case failures for the change and I will fix them as well.